tests/test_issue_464_cmdw_close_terminal_browser_split.py

```
@@ -26,7 +26,7 @@
 from cmux import cmux, cmuxError
 
 
-SOCKET_PATH = os.environ.get("CMUX_SOCKET", "/tmp/cmux-debug.sock")
+SOCKET_PATH = os.environ.get("CMUX_SOCKET_PATH", "/tmp/cmux-debug.sock")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug.sock"

(S108)